sui/src/bench.rs

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+#![deny(warnings)]
+
+use clap::*;
+
+use sui::benchmark::{bench_types, run_benchmark};
+use tracing::subscriber::set_global_default;
+use tracing_subscriber::EnvFilter;
+
+fn main() {
```

Comment:
What are the input parameters that this function is expecting?

Reply:
It uses CLI args. Am I supposed to specify something here?

Reply:
https://github.com/MystenLabs/sui/blob/b14b4a7c4c74675ca539ddd7fb27f2ba00007af8/sui/src/benchmark/bench_types.rs

Reply:
@asonnino https://github.com/MystenLabs/sui/blob/b14b4a7c4c74675ca539ddd7fb27f2ba00007af8/sui/src/benchmark/bench_types.rs